README.md

```diff
@@ -0,0 +1,135 @@
+# mobile-android-authentication
+
+Implementation of Authentication package
+
+## Installation
+
+To use Authentication in an Android Project:
+
+1. Add the following to the settings.gradle.kts
+
+```kotlin
+dependencyResolutionManagement {
+    ...
+    
+    repositories {
+        
+        ...
+        
+        maven("https://maven.pkg.github.com/govuk-one-login/mobile-android-authentication") {
+            if (file("${rootProject.projectDir.path}/github.properties").exists()) {
+                val propsFile = File("${rootProject.projectDir.path}/github.properties")
+                val props = Properties().also { it.load(FileInputStream(propsFile)) }
+                val ghUsername = props["ghUsername"] as String?
+                val ghToken = props["ghToken"] as String?
+
+                credentials {
+                    username = ghUsername
+                    password = ghToken
+                }
+            } else {
+                credentials {
+                    username = System.getenv("USERNAME")
+                    password = System.getenv("TOKEN")
+                }
+            }
+        }
+    }
+}
+```
+
+2. For local development, ensure you have a `github.properties` in the project's root which includes your username and an access token
```

Comment:
Is this file git ignored? Can we just add a warning here to specify that if adding an access token, it should never be checked in pls?

Reply:
It is git ignored, yeah